test/test.js

```diff
@@ -152,6 +157,52 @@ describe('lib', function() {
         .attribute('attribute3', ['s1', 's2'])
         .attribute('attribute4', {a:123, b:'STR'})
         .attribute('attribute5', [{a:1, b:'S1'}, {a:2, b:'S2'}])
+        .attribute('attribute6', true)
+        ._data.attributes).to.deep.equal(expected);
+    });
+
+    it('should add attribute property if boolean support', function() {
+      this.jedlik = new(this.lib)(true);
+
+      var expected = {
+        attribute1: {
+          value: 'STR',
+          type: 'S',
+          action: 'PUT'
+        },
+        attribute2: {
+          value: '1234',
+          type: 'N',
+          action: 'PUT'
+        },
+        attribute3: {
+          value: ['s1', 's2'],
+          type: 'SS',
+          action: 'PUT'
+        },
+        attribute4: {
+          value: { a: { N: '123' }, b: { S: 'STR' } },
+          type: 'M',
+          action: 'PUT'
+        },
+        attribute5: {
+          value: [ { M: { a: { N: '1' }, b: { S: 'S1' } } }, { M: { a: { N: '2' }, b: { S: 'S2' } } } ],
+          type: 'L',
+          action: 'PUT'
+        },
+        attribute6: {
+          value: 'true',
```

Comment:
I think we need this to be a boolean, rather than a string. I've just double checked what would happen if we passed `"BOOL: 'true'"` to ddb combined, we throw a validation error (see below)
![image](https://user-images.githubusercontent.com/52454994/132360174-7637b263-4087-425b-9e86-14089b9821e3.png)